osconfig_tests/test_suites/inventory/inventory.go

```diff
@@ -219,17 +255,17 @@ func runGatherInventoryTest(ctx context.Context, testSetup *inventoryTestSetup,
 	}
 
 	testCase.Logf("Checking inventory data")
-	// It can take a bit to start collecting data, retry a few times.
+	// It can take a bit to start collecting data, especially on Windows.
```

Comment:
Since this was an issue, might be worth defining "bit" which IIRC is on the order of 15 minutes.